<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="menu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>131</width>
     <height>381</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="button_set_machine_controls">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Movement</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>451</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="movement">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>411</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="moevement">
      <item>
       <layout class="QHBoxLayout" name="set">
        <item>
         <widget class="QLabel" name="label_step">
          <property name="text">
           <string>Step:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="step01mm">
          <property name="text">
           <string>0.1 mm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="step1mm">
          <property name="text">
           <string>1 mm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="step10mm">
          <property name="text">
           <string>10 mm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="step100mm">
          <property name="text">
           <string>100 mm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="movementspeed">
        <item>
         <widget class="QLabel" name="label_movement_speed">
          <property name="text">
           <string>Movement speed:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slider_movement_speed">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>80</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_movement_speed">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>80</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="engravespeed">
        <item>
         <widget class="QLabel" name="label_engrave_speed">
          <property name="text">
           <string>Engraving speed:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slider_engrave_speed">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>80</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_engrave_speed">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>80</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>239</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="button_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_apply">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider_movement_speed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_movement_speed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_movement_speed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_movement_speed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_engrave_speed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_engrave_speed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_engrave_speed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_engrave_speed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
